crates/utils/src/settings/structs.rs

```diff
@@ -49,6 +49,9 @@ pub struct Settings {
   #[default(None)]
   #[doku(example = "Some(Default::default())")]
   pub prometheus: Option<PrometheusConfig>,
+  #[default(None)]
+  #[doku(example = "*")]
+  pub cors_origin: Option<String>,
```

Comment:
Actually the example isnt that helpful, so there should also be a comment describing what cors is. Possibly with a link for further explanation.